tahoma_bringup/launch/tahoma_bringup.launch

```diff
@@ -39,8 +39,11 @@
     </include>-->
 
     <node name="fake_gripper_state" type="fake_gripper_state" pkg="tahoma_bringup" >
-      <param name="state" value="0.0" />
+      <param name="state" value="1.0" />
```

Comment:
We can stop faking this now that we can read the gripper joint status. The same node that publishes the status information should publish a joint state message with real information